crypto/Kconfig

```diff
@@ -3,6 +3,26 @@ menu "Crypto libraries for nRF5x SOCs."
 config NRFXLIB_CRYPTO
 	bool
 
+config CRYPTOCELL_CC310_USABLE
+	bool
+	depends on SOC_NRF52840 || (SOC_NRF9160 && (BUILD_WITH_TFM || !TRUSTED_EXECUTION_NONSECURE))
```

Comment:
Do we have any reason not to use CONFIG_TRUSTED_EXECUTION_SECURE here?

Reply:
Because we need to signal that this is usable even if we are in NS build to have the faux driver for entropy